core-web/libs/portlets/dot-content-drive/portlet/src/lib/shared/models.ts

```diff
@@ -1,32 +1,69 @@
 import { DotContentDriveItem, SiteEntity } from '@dotcms/dotcms-models';
 
+/**
+ * The status of the content drive.
+ *
+ * @export
+ * @enum {string}
+ */
 export enum DotContentDriveStatus {
     LOADING = 'loading',
     LOADED = 'loaded',
     ERROR = 'error'
 }
 
+/**
+ * The sort order of the content drive.
+ *
+ * @export
+ * @enum {string}
+ */
 export enum DotContentDriveSortOrder {
     ASC = 'asc',
     DESC = 'desc'
 }
 
+/**
+ * The pagination of the content drive.
+ *
+ * @export
+ * @interface DotContentDrivePagination
+ */
 export interface DotContentDrivePagination {
     limit: number;
     offset: number;
 }
```

Comment:
I think this going to be repeated a lot in other tables... 

I think we should make this agnostic to the tables not the content drives to avoid dupe.